charts/ziti-controller/values.yaml

```diff
@@ -28,22 +32,14 @@ clientApi:
     annotations: {}
     # -- deprecated: tls passthrough is required
     tls: {}
-  altIngress:
```

Comment:
delete alternative ingress because it's redundant and simpler to add a host rule to the existing Ingress